src/lib/components/file-browser/file-list/file-list.component.ts

```diff
@@ -595,4 +600,23 @@ export class TerraFileListComponent implements OnInit, AfterViewInit, OnChanges,
         let uploadPrefix: string = this.currentStorageRoot ? this.currentStorageRoot.key : '/';
         this.activeStorageService.uploadFiles(fileList, uploadPrefix);
     }
+
+    private deleteObjects(): void {
```

Comment:
gerne weiterhin mit unterstrich als prefix
```suggestion
    private _deleteObjects(): void {
```